pkg/controller/core.oam.dev/v1alpha2/core/components/componentdefinition/componetrevision_test.go

```diff
@@ -179,17 +181,10 @@ var _ = Describe("Test DefinitionRevision created by ComponentDefinition", func(
 			cd.Name = cdName
 			cd.Spec.Schematic.CUE.Template = fmt.Sprintf(cdTemplate, fmt.Sprintf("test-v%d", revisionNum))
 			Expect(k8sClient.Create(ctx, cd)).Should(BeNil())
-			reconcileRetry(&r, req)
-			revKey := client.ObjectKey{Namespace: namespace, Name: fmt.Sprintf("%s-v%d", cdName, revisionNum)}
-			revisionNum++
-			var defRev v1beta1.DefinitionRevision
-			Eventually(func() error {
-				return k8sClient.Get(ctx, revKey, &defRev)
-			}, 10*time.Second, time.Second).Should(BeNil())
```

Comment:
why deleted?

Reply:
This is align with the test in application, and this part of the logic has been moved to another location.